src/backend/joanie/tests/payment/base_payment.py

```diff
@@ -0,0 +1,33 @@
+"""Common Base Payment backend test"""
+from django.core import mail
+from django.test import TestCase
+
+
+class BasePaymentTestCase(TestCase):
+    """Common method to test the Payment Backend"""
+
+    def _check_purchase_order_email_sent(self, email, username, order):
```

Comment:
I think we could simply pass the order object then retrieve `email` and `username` from `order.owner`

Reply:
we could :-) but as explained in other PR, i like showing directly from the test what we are testing